opentelemetry-exporter-google-cloud/src/opentelemetry/exporter/cloud_monitoring/__init__.py

```diff
@@ -280,16 +292,34 @@ def export(
                     UNIQUE_IDENTIFIER_KEY
                 ] = self.unique_identifier
 
-            point = series.points.add()
-            if isinstance(record.aggregator, SumAggregator):
-                data_point = record.aggregator.checkpoint
-            elif isinstance(record.aggregator, ValueObserverAggregator):
-                data_point = record.aggregator.checkpoint.last
+            point_dict = {"interval": {}}
```

Comment:
@AndrewAXue provided some base code for this and he used this method - maybe he can provide some insight?

Reply:
There was no real reason why it was originally done using a proto. I just wasn't aware you could even do it this way.

Reply:
I prefer to use the the protos since they will surface validation errors where they are set. Created https://github.com/GoogleCloudPlatform/opentelemetry-operations-python/issues/56, lets go ahead with this for now